district_data_import.py

```diff
@@ -222,6 +222,17 @@
         parent2_email_field = 'SPOUSE_EMAIL' if district_record['Parent 1 Last Name'] else 'EMAIL'
         if district_record['Parent2Email'] and not dp_donorrecord[parent2_email_field]:
             dp_donorrecord[parent2_email_field] = district_record['Parent2Email']
+
+        #set homeschool IF needs to be changed by looking at all students for the donor
+        #by this time, all students have been promoted and set to their new school
+        all_students = dp.get_students_for_donor(dp_studentrecord['DONOR_ID'])
```

Comment:
I think this may miss students where some of the donor information is different, but it will match on import. Probably not a huge deal, but something to be aware of.

Reply:
you mean newly created donor?  Yes, it will miss those -- although I don't think it could be helped.  Maybe we should put out a msg in the output to be aware of these scenarios whenever new donors are added to check for these cases.  I know I will forget what all needs to be checked for these scenarios.